### PR TITLE
Add user stop words Madlib. #45

### PR DESCRIPTION
**Pull request by greebie**
This PR gives the user the options to use NLTK stopwords and/or add their own. This was a common demand from the datathon.

I also removed the International () function call because it is not doing what I originally thought it was doing.

- add set for user stopwords.
- remove misleading 'international' call until it's fixed.

### CODE REVIEW

**Comment by greebie** [comment]
Okay.  Will fix. 

I knew you were going to say that about international(), but I just couldn't bear seeing it just be wrong and out there in the open for such a small change.  Also, I knew that #42 was still in the issues, so the tracking is still available.

**Comment by greebie** [comment]
Latest commit should resolve all issues stated above.

**Comment by greebie** [comment]
I think that's a different issue which might be fixed more generally in another branch, especially if we look at making a module to cover the functions.

Essentially we have general madlibs and more specific madlibs and it's not always clear which is which. I don't think that it's that helpful to tell people to scroll up to change something. Let's look at the overall explanations / docs / comments together as a separate issue.  (Maybe after receiving feedback from the datathon as well.)

**Comment by ianmilligan1** [comment]
OK. I still don't think

```
vals = [x[1] for x in tokens if x[0] not in STOP_WORDS] # Filter based on STOP_WORDS (above).
```

is clear? Filter what? Filter in? Filter out? Can you try to take a kick at the can to make it a bit more usable?

**Comment by greebie** [comment]
Now that I've been able to look at this more closely, I see your point. I have some follow-up work this morning, but will get this fixed in the afternoon.

File: auk-notebook.ipynb [hunk 1]
```diff
@@ -645,7 +645,7 @@
    "source": [
     "## Overall Collection Characteristics\n",
     "\n",
-    "Change the variables in the following cell to manipulate the analysis you'll be running to understand overall collection characteristics. In this case, they mostly affect the visualization that we generate.\n"
+    "Change the variables in the following cell to manipulate the analysis you'll be running to understand overall collection characteristics. In this case, they mostly affect the visualization that we generate. You may wish to add or remove words to `STOP_WORDS_USER`in the user configuration cell to fine-tune the results.\n"
```
Comment (ianmilligan1) [inline auk-notebook.ipynb]: maybe `fine-tune the results` -> `remove words that may be overwhelming your analysis`

**Pull request merged**